app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -124,6 +135,43 @@ def get_images
         process_collection(images, :vms) { |image| parse_image(image) }
       end
 
+      def get_security_groups
+        security_groups = gather_data_for_this_region(@nsg)
```

Comment:
@bronaghs I think the security_groups method should be there, and line 139 could be removed. It's harmless, though.